src/components/OperatorChecklist.tsx

```diff
@@ -74,7 +75,6 @@ export function OperatorChecklist({ operatorId }: { operatorId: string | undefin
             </ChecklistItem>
             <Separator />
             <ChecklistItem
-                disabled
```

Comment:
I don't predict the `disabled` checklist items to be a thing in the future, thus I remove the feature entirely.